components/camel-salesforce/camel-salesforce-component/pom.xml

```diff
@@ -34,6 +34,9 @@
 
     <properties>
         <salesforce.component.root>..</salesforce.component.root>
+        <grpc.version>1.62.2</grpc.version>
+        <protobuf.version>3.21.5</protobuf.version>
+        <protobuf-maven-plugin.version>2.1.1</protobuf-maven-plugin.version>
```

Comment:
All of this properties should come from the parent pom and not redefined in this one.

Reply:
Good catch. Fixed.